src/Data/Pool/Internal.hs

```diff
@@ -64,15 +65,16 @@ data PoolConfig a = PoolConfig
   -- ^ The maximum number of resources to keep open across all stripes. The
   -- smallest acceptable value is @1@.
   --
-  -- /Note:/ for each stripe the number of resources is divided by the number of
-  -- stripes and rounded up, hence the pool might end up creating up to @N - 1@
-  -- resources more in total than specified, where @N@ is the number of stripes.
+  -- /Note/: Each stripe will try to have an equal amount of resources per
+  -- stripe. If the number of resources does not evenly divide into the
+  -- stripes, then some stripes will have one fewer resource than others.
   , poolNumStripes :: !(Maybe Int)
-  -- ^ The number of stripes in the pool.
+  -- ^ The number of stripes in the pool. If 'Nothing' is provided, then
+  -- this will use 'getNumCapabilities'.
   --
-  -- /Note:/ if set to 'Nothing', it defaults to the number of capabilities,
-  -- which ensures that threads never compete over access to the same stripe and
-  -- results in a very good performance in a multi-threaded environment.
+  -- The resource count in 'poolMaxResources' will be split evenly among
+  -- the available stripes. If there are fewer stripes than resources, then
+  -- this will only create as many stripes as resources.
```

Comment:
It appears swapped.
```suggestion
  -- The resource count in 'poolMaxResources' will be split evenly among
  -- the available stripes. If there are fewer resources than stripes, then
  -- this will only create as many stripes as resources.
```